shopping-cart/shopping-cart-scala/shopping-cart-api/src/main/scala/com/example/shoppingcart/api/ShoppingCartService.scala

```diff
@@ -1,5 +1,7 @@
 package com.example.shoppingcart.api
 
+import java.time.OffsetDateTime
```

Comment:
Prefer `java.time.Instant`. `OffsetDateTime` (IMHO) provides little value compared to `Instant` and `ZonedDateTime` and, in this particular code base, introduces confusion: there are enough `offsetFoo` flying around on a read side processor.

Reply:
I will change